src/components/Login/index.jsx

```diff
@@ -66,34 +74,34 @@ export default class Login extends React.Component {
               display: "flex",
               flexDirection: "column",
               justifyContent: "center",
-              alignItems: "center"
+              alignItems: "center",
             }}
           >
             <div
               className="login-title"
               style={{
-                fontWeight: "bold"
+                fontWeight: "bold",
               }}
             >
               SendGrid
             </div>
             {isShowingError && (
               <div style={{ color: "#b71c1c", marginTop: "1rem" }}>
-                Incorrect Credentials
+                {errorMessage}
```

Comment:
actually just noticed your other potential error message "Incorrect credentials". I'm guessing determining when to show that message will be in the other PR since I saw some validation stuff there. you can ignore the above comment for now